ios/engine/KMEI/KeymanEngine/Classes/KeymanWebViewDelegate.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  KeymanWebViewDelegate.swift
+//  KeymanEngine
+//
+//  Created by Gabriel Wong on 2017-10-17.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+public protocol KeymanWebViewDelegate: class {
+  func updatedFragment(_ fragment: String)
```

Comment:
New class?

Reply:
It was originally declared in `KMManager+Internal.h`